memgpt/persistence_manager.py

```diff
@@ -133,11 +131,11 @@ def json_to_message(self, message_json) -> Message:
             tool_calls = None
 
         return Message(
-            user_id=self.config.anon_clientid,
-            agent_id=self.agent_config.name,
+            user_id=self.agent_state.user_id,
+            agent_id=self.agent_state.id,
             role=message["role"],
             text=message["content"],

```

Comment:
@sarahwooders Also, I think we should be consistent with OAI and make the naming `text` -> `content` in the data type.

Relatedly, in the OAI API spec, UserMessage is slightly different than a regular Message, in that `Message.content` can be either a string (as per usual), or an array. This is to support multi-modality:
<img width="559" alt="image" src="https://github.com/cpacker/MemGPT/assets/5475622/d5226460-1d5f-47c4-9de0-ce217e2aa86d">

This is something we might want to consider baking into the schema on the initial refactor release (0.2.12).